core/store/src/config.rs

```diff
@@ -240,17 +240,22 @@ impl Default for StoreConfig {
             block_size: bytesize::ByteSize::kib(16),
 
             trie_cache: TrieCacheConfig {
-                default_max_bytes: DEFAULT_SHARD_CACHE_TOTAL_SIZE_LIMIT,
+                default_max_bytes: 500_000_000,
                 // Temporary solution to make contracts with heavy trie access
                 // patterns on shard 3 more stable. It was chosen by the estimation
                 // of the largest contract storage size we are aware as of 23/08/2022.
                 // Consider removing after implementing flat storage. (#7327)
                 // Note: on >= 1.34 nearcore version use 1_000_000_000 if you have
                 // minimal hardware.
-                per_shard_max_bytes: HashMap::from_iter([(
-                    ShardUId { version: 1, shard_id: 3 },
-                    3_000_000_000,
-                )]),
+                per_shard_max_bytes: HashMap::from_iter([
+                    (ShardUId { version: 1, shard_id: 3 }, 3_000_000_000),
+                    // After resharding "token.sweat" account moves to shard 4
+                    (ShardUId { version: 2, shard_id: 4 }, 3_000_000_000),
+                    // Shard 1 is dedicated to aurora and it had very few cache
+                    // misses even with cache size of only 50MB
+                    (ShardUId { version: 1, shard_id: 1 }, 50_000_000),
+                    (ShardUId { version: 2, shard_id: 1 }, 50_000_000),
```

Comment:
Isn't it the default now? Does it need to be set explicitly?

Reply:
nope, I've changed the default to 500MB